commons/src/main/java/fr/free/nrw/commons/MediaWikiImageView.java

```diff
@@ -151,7 +151,7 @@ private void loadImageIfNecessary(final boolean isInLayoutPass, final boolean tr
         // from the network.
         ImageContainer newContainer = mImageLoader.get(mUrl,
                 new ImageListener() {
-                    @Override
```

Comment:
In my opinion, it's a loss to remove them. I opened up Android Studio and saw the red squiggles. When I select one and press alt + enter, it gave me an option to bump the language level. I did that and now the squiggles are gone.